web/src/app/shared/shared.module.ts

```diff
@@ -103,7 +104,7 @@ const DECLARATIONS = [
 ];
 
 @NgModule({
-  declarations: DECLARATIONS,
+  declarations: [DECLARATIONS, GermanTimeFormatPipe],
```

Comment:
suggestion: add GermanTimeFormatPipe to DECLARATIONS
but can also be done later

Reply:
fixed 👍🏻